mbed_project/mbed_program.py

```diff
@@ -73,11 +71,11 @@ def from_remote_url(cls, url: str, dst_path: Path) -> "MbedProgram":
                 "should cd to a directory containing a program before performing any other operations."
             )
 
-        mbed_os = MbedOS.from_new(dst_path / MBED_OS_DIR_NAME)
-        return cls(repo, program, mbed_os)
+        mbed_os = MbedOS.from_existing(dst_path / MBED_OS_DIR_NAME)
```

Comment:
This seems to have changed from `from_new` to `from_existing`, is that right?

Reply:
Yeah, `from_new` was incorrect as we're importing an existing program from a url here.

Reply:
Ah right, that explains it 👍